tests/click_tests/test_passphrase_mercury.py

```diff
@@ -329,3 +337,16 @@ def test_cycle_through_last_character(
         passphrase = DA_49 + "i"  # for i we need to cycle through "ghi" three times
         input_passphrase(debug, passphrase)
         enter_passphrase(debug)
+
+
+@pytest.mark.setup_client(passphrase=True)
+def test_last_char_timeout(device_handler: "BackgroundDeviceHandler"):
```

Comment:
comments talk about "digits" and "PIN" instead of "characters" and "Passphrase"

Reply:
https://github.com/trezor/trezor-firmware/pull/4385/commits/ba52fdf421ce16743174d0f20aafd7f4b1186c3b